docs/basics/functions.md

```diff
@@ -665,6 +669,110 @@ Which would return the following:
 "metrics": []
 ```
 
+### sample_include_filter
+
+| Applies to | Description |
+| :--------- | :---------- |
+| API | Only creates the sample if both a key and value is found in the sample.|
+|| In the absence of `sample_include_filter`, it defaults to include all.|
```

Comment:
I don't think this line is necessary.

Reply:
@haihongren What's the rationale for keeping these lines? If `sample_include_filter` is not used, of course all are included.